app/models/concerns/host_boot_files_extensions.rb

```diff
@@ -0,0 +1,32 @@
+# This module provides a method that exposes parameter information for a given host.
+# The hash is extensible, so any plugin/module can add a provider that will add its
+# information to the method.
+# Usage:
+# host = Host::Managed.find(my_host_id)
+# host.info
+# => A hash of hashes that describes all parameters associated with the host.
+#
+# Writing an extension:
+# Inherit HostInfo::Provider and override #host_info method and register the
+# provider by calling HostInfo.register_info_provider
+module HostBootFileExtensions
```

Comment:
Seems like some residual: I don't see this being used anywhere.

Reply:
LOL I was digging in the codebase for like 40 minutes trying to figure out how *this* works :-)